content/en-us/build/stateless/language-basic.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Language Basics"
+weight: 1002
+menu:
+  build:
+    parent: "phat-stateless"
+---
+
+The Phat Contract is written in Ink! language, which is extended from Rust language (more language supports like Python and Nodejs will be provided in the future). It is finally compiled to WebAssembly (WASM), which also takes security into consideration when designed, then you can upload and deploy them.
```

Comment:
Let's not promote Python and node.js in this wiki. It's too far away from an implementation, and we have no clue how it will work

Reply:
Why mentioning "takes security into consideration when designed" here?

Reply:
removed.